backend/src/apps/api/v1/urls.py

```diff
@@ -30,5 +30,9 @@
         "organizations/<int:organization_id>/documents/",
         include("apps.api.v1.documents"),
     ),
+    path(
+        "organizations/<int:organization_id>/ideas/",
+        include("apps.api.v1.ideas"),
+    ),
```

Comment:
Так как мы не зависим от организации, то и в пути ее следует убрать
```suggestion
    path(
        "ideas/",
        include("apps.api.v1.ideas"),
    ),
```